plugins/com.google.cloud.tools.eclipse.login/src/com/google/cloud/tools/eclipse/login/ui/AccountSelector.java

```diff
@@ -128,9 +130,10 @@ public int selectAccount(String email) {
       combo.select(index);
       selectedAccount = (Account) combo.getData(email);
     }
-    // Need to fire even if nothing is selected (which is an error state), to show the error
-    // message: https://github.com/GoogleCloudPlatform/google-cloud-eclipse/pull/1303
-    fireSelectionListeners();
+
+    if (oldIndex != index) {
+      fireSelectionListeners();
+    }
```

Comment:
It's related, but this PR is basically blocked on #1656. This will be cleaned up once #1656 is fixed.